src/signals/incident-management/containers/IncidentSplitContainer/components/IncidentSplitFormIncident/index.js

```diff
@@ -27,10 +27,18 @@ const IncidentSplitFormIncident = ({ parentIncident, subcategories, register })
     []
   );
 
+  const incidentSplitFormIncidentRefs = useMemo(() => Array(splitCount).fill().map(() => createRef()), [splitCount]);
+
+  useEffect(() => {
+    if (splitCount === 1) return;
+
+    incidentSplitFormIncidentRefs[splitCount - 1].current.scrollIntoView({ behavior: 'smooth' });
```

Comment:
Woudn't just one ref be enough? The request was just to scroll to the last added element.

Reply:
Changed it to a single ref